contrib/opencensus-ext-requests/opencensus/ext/requests/trace.py

```diff
@@ -93,6 +102,15 @@ def call(url, *args, **kwargs):
 
 def wrap_session_request(wrapped, instance, args, kwargs):
     """Wrap the session function to trace it."""
+    try:
+        # Check if request was sent from an exporter. If so, do not wrap.
+        disable_from_exporter = RuntimeContext.is_exporter_thread
+        if disable_from_exporter:
```

Comment:
Minor, would it be better to put `if RuntimeContext.is_exporter_thread:` directly?

Reply:
This is true.